base/docs/helpdb/Base.jl

```diff
@@ -972,7 +1037,15 @@ setdiff!
 """
     copysign(x, y)
 
-Return `x` such that it has the same sign as `y`
+Return `x` such that it has the same sign as `y`.
```

Comment:
this is worth rewording, if it has to change the sign then it's not returning `x` is it